packages/react-hooks/src/useScrollIntoView/useScrollIntoView.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { renderHook } from "@testing-library/react-hooks";
+import { useScrollIntoView } from "./useScrollIntoView";
+
+const expectAfterWait = (timeout: number, expectFn: () => void) =>
+    new Promise((resolve) => setTimeout(() => resolve(expectFn()), timeout));
+
+const scrollIntoView = jest.fn();
+const ref = {
+    current: {
+        scrollIntoView,
+    },
+};
+
+beforeEach(scrollIntoView.mockReset);
+
+test("should call scrollIntoView fn by itself", async () => {
+    // @ts-ignore
+    renderHook(() => useScrollIntoView({ ref }));
+
+    await expectAfterWait(0, () => expect(scrollIntoView.mock.calls.length).toBe(1));
```

Comment:
Dette ser kanskje litt rart ut, men er fordi effekten pakker scrollfunksjonen inn i en setTimeout, dermed havner funksjonen bak i callstacken, så jeg lager en setTimeout her også for at expecten havner på rett sted i callstacken.

Reply:
Hehe, Jest-versjonen av `setTimeout(myFunction, 0)` altså 😊